src/components/popups/PopupGlossaryTermSelect.tsx

```diff
@@ -25,16 +26,21 @@ export const PopupGlossaryTermSelect = ({wide, codemirror}: { wide?: boolean, co
     const [glossaryTermText, setGlossaryTermText] = useState<string>();
     const [glossaryTerm, setGlossaryTerm] = useState<Item<string> | undefined>();
     const [isInlineTerm, setIsInlineTerm] = useState<boolean>(true);
+    const [isTitledTerm, setIsTitledTerm] = useState<boolean>(isAda); // Ada should default to being checked
 
     const generateAndInsertGlossaryTerm = useCallback(() => {
         if (glossaryTerm) {
             const trimmedGlossaryTermText = glossaryTermText?.trim();
-            codemirror.current?.view?.dispatch(
-                codemirror.current?.view?.state.replaceSelection(`[glossary${isInlineTerm ? "-inline" : ""}:${glossaryTerm.value}${isInlineTerm && trimmedGlossaryTermText ? ` "${trimmedGlossaryTermText}"` : ""}]`)
+            codemirror.current?.view?.dispatch(codemirror.current?.view?.state.replaceSelection(`[glossary${isInlineTerm ? "-inline" : ""}${isTitledTerm ? "-titled" : ""}:${glossaryTerm.value}${isInlineTerm && trimmedGlossaryTermText ? ` "${trimmedGlossaryTermText}"` : ""}]`)
```

Comment:
Actually, the real problem is that "-titled" should only be added when both `isTitledTerm` and `isInlineTerm` is `true`. Currently if an Ada content editor creates a non-inlined term the generated text will be the incorrect `[glossary-titled:...]` because `isTitledTerm` is true by default for Ada.
With that change we can undo the previous commit as that complicates things unnecessarily.